client/interfaces.go

```diff
@@ -425,6 +426,32 @@ type InstanceServer interface {
 	UpdateWarning(UUID string, warning api.WarningPut, ETag string) (err error)
 	DeleteWarning(UUID string) (err error)
 
+	// Authorization functions
+	GetGroupNames() (groupNames []string, err error)
```

Comment:
`*Group*()` functions are too vaguely named. 
Lets use AuthGroup to align with API and DB.